_delphi_utils_python/delphi_utils/covidcast_wrapper.py

```diff
@@ -0,0 +1,251 @@
+"""module for covidcast api call wrapper."""
+
+from datetime import date, timedelta
+from typing import Iterable, Union
+
+import pandas as pd
+from delphi_epidata import Epidata
+from epiweeks import Week
+
+
+def date_generator(startdate: date, enddate: date) -> Iterable[date]:
+    """
+    Take start date and end date and generates date string.
+
+    Parameters
+    ----------
+    startdate: date
+    enddate: date
+
+    Returns
+    -------
+    generator of str
+    """
+    while startdate <= enddate:
+        yield startdate.strftime("%Y-%m-%d")
+        startdate = startdate + timedelta(days=1)
+
+
+def _parse_datetimes(date_int: int, time_type: str, date_format: str = "%Y%m%d") -> Union[pd.Timestamp, None]:
+    """Convert a date or epiweeks string into timestamp objects.
+
+    Datetimes (length 8) are converted to their corresponding date, while epiweeks (length 6)
+    are converted to the date of the start of the week. Returns nan otherwise
+
+    Epiweeks use the CDC format.
+
+    :param date_int: Int representation of date.
+    :param date_format: String of the date format to parse.
+    :returns: Timestamp.
+    """
+    date_str = str(date_int)
+    if time_type == "day":
+        return pd.to_datetime(date_str, format=date_format)
+    if time_type == "week":
+        epiwk = Week(int(date_str[:4]), int(date_str[-2:]))
+        return pd.to_datetime(epiwk.startdate())
+    return None
+
+
+def metadata() -> Union[pd.DataFrame, None]:
+    """
+    Make covidcast metadata api call.
+
+    Returns
+    -------
+    pd.DataFrame of covidcast metadata.
+    """
+    # pylint: disable=W0212
+    response = Epidata._request("covidcast_meta")
```

Comment:
```suggestion
    response = Epidata.covidcast_meta()
```
it does ~exactly the same thing, but you might as well use the public method; see https://github.com/cmu-delphi/delphi-epidata/blob/8746ff2ef7a936bb93628bc1358471d7c6c4f5f8/src/client/delphi_epidata.py#L611-L615

Reply:
Ah I missed the public method for metadata for some reason. Making change